server/mlabns/tests/test_tool_fetcher.py

```diff
@@ -40,45 +68,51 @@ def insertSliverTool(self, tool_id, site_id=None, status_ipv4=None,
         tool.latitude = latitude
         tool.longitude = longitude
         tool.country = country
-        tool.put()
+        self.created_tools.append(tool)
```

Comment:
So each combination of site and country and IPv4/v6 will have its own Sliver Tool? Where is the code for tools initialization in the real case?